connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -72,8 +73,8 @@
     private final OffsetBackingStore offsetBackingStore;
     private final Map<String, Object> producerProps;
 
-    private HashMap<String, WorkerConnector> connectors = new HashMap<>();
-    private HashMap<ConnectorTaskId, WorkerTask> tasks = new HashMap<>();
+    private ConcurrentMap<String, WorkerConnector> connectors = new ConcurrentHashMap<>();
```

Comment:
Mentioned previously, but since these fields are accessed concurrently, they should be final.